snippets/quantity-input.liquid

```diff
@@ -16,10 +16,10 @@
     {% render 'icon-minus' %}
   </button>
   <input
-    class="quantity__input"
+    class="quantity__input no-js-hidden"
```

Comment:
The idea is to have that re-used everywhere we have a qty input. I only added it just to the variant-row to avoid increasing the scope of testing for now